obspy/taup/tau.py

```diff
@@ -346,17 +346,15 @@ def plot_rays(self, phase_list=None, plot_type="spherical",
 
             intp = matplotlib.cbook.simple_linear_interpolation
             radius = self.model.radius_of_planet
+            phase_names_encountered = set([ray.name for ray in arrivals])
```

Comment:
Can use a set literal.
```suggestion
            phase_names_encountered = {ray.name for ray in arrivals}
```

Reply:
Oh OK, didn't realize that was in 2.7